pkg/cmd/pulumi/state_setignorechanges.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2024, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pulumi/pulumi/pkg/v3/resource/deploy"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/resource"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/util/cmdutil"
+
+	"github.com/spf13/cobra"
+)
+
+func newStateSetIgnoreChangesCommand() *cobra.Command {
+	var stack string
+	var paths []string
+	var yes bool
+
+	cmd := &cobra.Command{
+		Use:   "set-ignore-changes [resource URN] --path [property-path-1] ...",
```

Comment:
```suggestion
		Use:   "set-ignore-changes [resource URN] [--path property-path-1] ...",
```

Also maybe this could be `pulumi state set ignore-changes`, so we can have other `pulumi state set` commands later potentially?   I think that would maybe fit better with the current design?